rate-limiting.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Rate limiting
+description: "What is rate limiting and how to deal with it."
+icon: "gauge-max"
+---
+
+Whenever the octomind agent is discovering your page or the test runner is executing test cases it is sending requests to your page (or your app).
+
+To ensure a well defined behavior all requests are send through one fixed IP address: `35.192.162.70`
+
+## Agent discovery
+
+Agent discovery happens as if a human would use your web application, however most of the time the agent interacts with the website slightly faster than a human would.
+There might also be more than one discovery run in parallel.
+
+## Test runner
+
+When you "run all" testcases or a scheduled run occurs on a given test target all activated test cases are run at once and with up to 20 parallel browser sessions.
+This can lead to either an overload of your application or a rate limiter getting active, if you have one in place.
+
+## Login / Authentication
+
+If your application requires a login the configured test user is used for the agent discovery as well as for all the test runs. This could also lead to 
```

Comment:
```suggestion
If your application requires a login the configured test user is used for the agent discovery as well as for all the test runs. This could also lead to **hitting a limit of allowed logins**, depending on your login system.
```